recipes/conda/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "4.0.6" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  fn: conda-{{ version }}.tar.gz
+  url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
+  sha256: 4ca7ec875de13519aeb3eebc9beca23adf3c41e8852c186deb503e8bc2282c40
+
+build:
```

Comment:
From the [conda repository](https://github.com/ContinuumIO/anaconda-recipes/blob/master/conda-env/meta.yaml)

```
  # These are present when the new environment is created
  # so we have to exempt them from the list of initial files
  # for conda-build to realize they should be included.
  always_include_files:
    - bin/conda                 [unix]
    - bin/activate              [unix]
    - bin/deactivate            [unix]
    - Scripts/activate.bat      [win]
    - Scripts/deactivate.bat    [win]
```